.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no -- commitlint --edit $1
```

Comment:
Can we make this an npm script instead of `npx`, that way it can be a first class citizen 😄 

e.g. `npm run commitlint $1`